internal/condition/condition.go

```diff
@@ -26,4 +26,8 @@ func OnCI() bool {
 
 func BuiltViaCI() bool {
 	return constants.OnCI == "true"
-}
\ No newline at end of file
+}
+
+func InStateSvc() bool {
+	return strings.HasSuffix(os.Args[0], constants.StateSvcCmd)
```

Comment:
Should use `osutils.ExecutableName()`. But given my missconception about where the notification was sent I'm ok with just dropping this altogether. The state-svc will never run any runner code.